api/tools/src/main/resources/bigquery/es_person.sql

```diff
@@ -41,6 +77,102 @@ LEFT JOIN (
     1) co
 ON
   co.person_id = p.person_id
+LEFT JOIN (
+  SELECT
+    d.person_id person_id,
+    ARRAY_AGG(DISTINCT drug_concept_id) drug_concept_ids,
+    ARRAY_AGG(DISTINCT drug_source_concept_id) drug_source_concept_ids,
+    ARRAY_AGG(STRUCT( drug_concept_id AS concept_id,
+        drug_source_concept_id AS source_concept_id,
+        drug_exposure_start_date AS start_date,
+        DATE_DIFF(drug_exposure_start_date, DATE(p.YEAR_OF_BIRTH, p.MONTH_OF_BIRTH, p.DAY_OF_BIRTH), YEAR) AS age_at_start_date,
+        v.visit_concept_id,
+        CAST(NULL as FLOAT64) as value_as_number,
+        CAST(NULL as INT64) as value_as_concept_id)) drugs
+  FROM
+    `{BQ_DATASET}.drug_exposure` d
+  LEFT JOIN
+    `{BQ_DATASET}.person` p
+  ON
+    d.person_id = p.person_id
+  LEFT JOIN
+    `{BQ_DATASET}.visit_occurrence` v
+  ON
+    d.visit_occurrence_id = v.visit_occurrence_id
+  WHERE
+    MOD(p.person_id, {PERSON_ID_MOD}) = 0
+  GROUP BY
+    1) d
+ON
+  d.person_id = p.person_id
+LEFT JOIN (
+  SELECT
+    pr.person_id person_id,
+    ARRAY_AGG(DISTINCT procedure_concept_id) procedure_concept_ids,
+    ARRAY_AGG(DISTINCT procedure_source_concept_id) procedure_source_concept_ids,
+    ARRAY_AGG(STRUCT( procedure_concept_id AS concept_id,
+        procedure_source_concept_id AS source_concept_id,
+        procedure_date AS start_date,
+        DATE_DIFF(procedure_date, DATE(p.YEAR_OF_BIRTH, p.MONTH_OF_BIRTH, p.DAY_OF_BIRTH), YEAR) AS age_at_start_date,
+        v.visit_concept_id,
+        CAST(NULL as FLOAT64) as value_as_number,
+        CAST(NULL as INT64) as value_as_concept_id)) procedures
+  FROM
+    `{BQ_DATASET}.procedure_occurrence` pr
+  LEFT JOIN
+    `{BQ_DATASET}.person` p
+  ON
+    pr.person_id = p.person_id
+  LEFT JOIN
+    `{BQ_DATASET}.visit_occurrence` v
+  ON
+    pr.visit_occurrence_id = v.visit_occurrence_id
+  WHERE
+    MOD(p.person_id, {PERSON_ID_MOD}) = 0
+  GROUP BY
+    1) pr
+ON
+  pr.person_id = p.person_id
+LEFT JOIN (
+  SELECT
+    m.person_id person_id,
+    ARRAY_AGG(DISTINCT measurement_concept_id) measurement_concept_ids,
+    ARRAY_AGG(DISTINCT measurement_source_concept_id) measurement_source_concept_ids,
+    ARRAY_AGG(STRUCT( measurement_concept_id AS concept_id,
+        measurement_source_concept_id AS source_concept_id,
+        measurement_date AS start_date,
+        DATE_DIFF(measurement_date, DATE(p.YEAR_OF_BIRTH, p.MONTH_OF_BIRTH, p.DAY_OF_BIRTH), YEAR) AS age_at_start_date,
+        v.visit_concept_id,
+        value_as_number,
```

Comment:
@freemabd1 please double check this - added the value_as_number/concept

Reply:
Looks good to me. Also, to cleanup data that is invalid/incorrectly coded should each join to domain table include all source or standard concept ids that match concept ids in the concept table?

Reply:
Yikes - is that a real thing? Let me file a ticket to fix this in a follow-up; if I'm understanding correctly, it shouldn't cause any bugs (yet) to have invalid data here but it will bloat the index size.

Filed https://precisionmedicineinitiative.atlassian.net/browse/RW-2242